props/saber_fett263_buttons.h

```diff
@@ -1823,17 +1828,17 @@ SaberFett263Buttons() : PropBase() {}
           Event(BUTTON_NONE, EVENT_SWING);
         }
       }
-      // EVENT_THRUST
-      if (mss.y * mss.y + mss.z * mss.z < 16.0 &&
-          mss.x > 14  &&
-          fusor.swing_speed() < 150) {
-        if (millis() - thrust_begin_millis_ > 15) {
-          Event(BUTTON_NONE, EVENT_THRUST);
-          thrust_begin_millis_ = millis();
-        }
-      } else {
+    }
```

Comment:
This can't be right, can it?
Does this compile?

Reply:
Yes, that closes Line 1774, just double checked with and without the new define and it compiles fine.  This moves Thrust detection outside "if (SaberBase::IsOn()) {...} for use while Off.

Reply:
github is not showing line 1774 as new or changed in this PR though.
So either github is bonkers, or something is wrong here. (or both I guess)
In my diff I don't see any new/changed lines with a "{" in it anywhere before this line.

Reply:
Ok, now It's showing me the rest of the lines below!
before it was only showing this one line
github IS bonkers :(